app/assets/stylesheets/views/_cookie-settings.scss

```diff
@@ -0,0 +1,4 @@
+.cookie-settings__confirmation {
+  @include core-19;
```

Comment:
Is there a reason for not using the new mixins? `@include govuk-font(19);`

Reply:
@alex-ju No, just failing to break out of old habits! Will correct 👍